mp/src/game/shared/basecombatweapon_shared.h

```diff
@@ -247,6 +247,7 @@ class CBaseCombatWeapon : public BASECOMBATWEAPON_DERIVED_FROM
 	
 	virtual bool			CanHolster( void ) const { return TRUE; };		// returns true if the weapon can be holstered
 	virtual bool			DefaultDeploy( char *szViewModel, char *szWeaponModel, int iActivity, char *szAnimExt );
+    virtual float           DeployTime() const { return 0.0f; }
```

Comment:
Perhaps 0 deploy time may not be best for every weapon now that I think about it. Just make 0.5 the default so people can't spam the paintgun or other weapons in lobbies.

Leave it as a virtual function though so anything can override it.

Reply:
Consider it done